inc/updater/namespace.php

```diff
@@ -54,20 +54,24 @@ function init() {
 		$theme_id = get_file_data( $theme_path . $file . '/style.css', [ 'ThemeID' => 'Theme ID' ] )['ThemeID'];
 
 		if ( ! empty( $theme_id ) ) {
-			$package_arr[] = $theme_path . $file . '/style.css';
+			$packages['themes'][] = $theme_path . $file . '/style.css';
 		}
 	}
 
-	run( $package_arr );
+	return $packages;
 }
 
 /**
  * Run Git Updater Lite for potential packages.
  *
  * @return void
  */
-function run( $package_arr ) {
-	foreach ( $package_arr as $package ) {
+function run() {
```

Comment:
Is there any scenario where we may want to call `run( $packages )`? Maybe for tests?

If so, it might be worth having:

```php
function bootstrap() {
    add_action( 'init', __NAMESPACE__ . '\\init' );
}

function init() {
    run( get_packages() );
}

function get_packages() {
    ...
}

function run( $packages ) {
    ...
}
```

Reply:
Is that over complicating things? I'm having a difficult time understanding what you're trying to solve there.

Reply:
I was just noticing that previous, `run()` could be called from anywhere using any packages, which may come in useful at another point during runtime, and having `get_packages()` and `run()` being independent of each other may be best.

Since the PR removes the parameter from `run()`, they're now tightly coupled, which also means tests can't independently verify each function as well.

Reply:
Keeping an open mind and looking at it again. If there was more going on, then there might be more benefit in separating them. But there's not much going on in `run()` that someone couldn't achieve in a few lines.

Since this was more a thought and not a blocker, I think it's fine to keep as-is and if we want to open stuff up later or there's an issue in testing, we can revisit it.